admin/python/lsst/qserv/admin/qservAdmin.py

```diff
@@ -486,7 +494,9 @@ def _createTable(self, options, dbName, tableName):
     def getNode(self, nodeName):
         """
         Returns node definition for specified node name. Returned dictionary has
-        keys defined in https://dev.lsstcorp.org/trac/wiki/db/Qserv/CSS#Node-related
+        keys defined in https://dev.lsstcorp.org/trac/wiki/db/Qserv/CSS#Node-related.
+        The state of the node (which does not have separate key in CSS) is returned
+        as 'state' key in the dictionary.
```

Comment:
There is also a css doc here: http://www.slac.stanford.edu/exp/lsst/qserv/2015_02/devel/css.html
All should be in the same place.
I prefer using sphinx so that the doc is up to date with the code, which is not true for Trac.
Is it possible to move merge the trac css doc with the one in sphinx?